src/transformer/html.ts

```diff
@@ -119,8 +119,7 @@ export class HTMLTransformer {
     const { idGenerator, mediaPathGenerator } = options
 
     this.modelMap = modelMap
-    const manuscript = findManuscript(this.modelMap)
-    this.labelTargets = buildTargets(fragment, manuscript)
+    this.labelTargets = buildTargets(fragment)
```

Comment:
@mnatsheh, I removed manuscript here as the second argument because this is the only reason we are passing MPManuscript to body-editor from article editor and the last reason we need MPManuscript on FE. However, I don't think we actually use labels customisation on FE. Please advise.

Reply:
I'm OK with removing this.